lib/jekyll/commands/new.rb

```diff
@@ -34,7 +35,7 @@ def process(args, options = {})
             create_site new_blog_path
           end
 
-          Jekyll.logger.info "New jekyll site installed in #{new_blog_path}."
+          Jekyll.logger.info "New jekyll site \"#{new_blog_title}\" installed in #{new_blog_path}."
```

Comment:
test.fmt is raising offence warnings for %Q{ } usage. So switching over to nested single quotes,